src/styles/colors.ts

```diff
@@ -1,12 +1,21 @@
 const COLORS = {
-  attorneyColor: '#85b4d2',
-  abaBlue: '#0069A9',
+  // Blues
+  primaryBlue: '#0069A9', // tearful-blue
+  secondaryBlue: '#035D85', // grieving
+  tertiaryBlue: '#003453', // drowning
+  lightPrimaryBlue: '#CCE1EE',
+  lighterPrimaryBlue: '#E8F6FF',
+  // Greys
+  lightGrey: '#F2F2F2',
+  midGey: '#CCC',
+  midDarkGrey: '#818181',
+  darkGrey: '#565656',
   dark: '#292929',
```

Comment:
To stay consistent, can we call these `primaryGrey`, `secondaryGrey` etc?
I think we should either do this or call other colors `lightBlue`, `midBlue` etc